### PR TITLE
[GG-777] Use SVG for icons

### PR DESCRIPTION
**Pull request by VilhelmNielsen**
This PR replaces _most of_ the icons provided by the CPH theme from icon font to inline SVG.

Also snuck in an a11y improvement in 918f9241750dc8dcc9ce0c3255f53d28311cc6ba

cc @zendesk/guide-growth 

JIRA: https://zendesk.atlassian.net/browse/GG-777

### CODE REVIEW

File: styles/_collapsible-nav.scss [hunk 1]
```diff
@@ -37,8 +37,12 @@
   }
 
   &::after {
-    @extend .icon;
-    content: "\25BE";
+    content: '';
+    background-size: cover;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' focusable='false' viewBox='0 0 16 16'%3E%3Cpath fill='none' stroke='currentColor' stroke-linecap='round' d='M4 6.5l3.6 3.6c.2.2.5.2.7 0L12 6.5'/%3E%3C/svg%3E");
```
[the remark this answers is not in the document]
Comment (VilhelmNielsen) [reply styles/_collapsible-nav.scss]: Sure, the only nit I have with that, is that the fill and stroke colors are baked into the data uri, so hypothetically we could end up with:

```scss
$svg_lock = ...
$svg_lock_yellow = ...
$svg_lock_slightly-less-yellow = ...
...
```

[the remark this answers is not in the document]
Comment (augustocravosilva) [reply styles/_collapsible-nav.scss]: Hum, isn't the color suppose to change with the theme color that you apply in the settings panel?

[the remark this answers is not in the document]
Comment (VilhelmNielsen) [reply styles/_collapsible-nav.scss]: That's not doable when inserting SVG with CSS, only when they're inlined in the markup. We'd have to do some engineering with our quirky compiler, and it's not even supported in sass without injecting some magic.

**Comment by TetianaG** [comment]
@VilhelmNielsen I have found a few missing icons:
- search icon in community, search widget, and requests page
- dropdown icon in language dropdown, and sorting (examples: Community topics on Community page, sort by recent in My Profile, sort by comments in Article/Post page)

I am not sure that right dropdown icons are used on Activities page, and new Request page.

**Comment by VilhelmNielsen** [comment]
@TetianaG Thanks, I've added icons in the missing places you mentioned

> I am not sure that right dropdown icons are used on Activities page, and new Request page.
If you mean the responsive dropdowns, those icons are correct.

**Comment by TetianaG** [comment]
@VilhelmNielsen 
There are still few issues:
- search icon in the widget on category, section, article pages, my requests page
- settings icon - in community post, comments (articles, posts)
- missing dropdown logo - All community posts in hc/en-us/community/posts
- cross icons on New request page for attachments are not aligned with the file name
- no indicators of mandatory fields on New Request Form
- missing dropdown logo for Sort by recent activity across user profile pages

**Comment by VilhelmNielsen** [comment]
@TetianaG 
> * search icon in the widget on category, section, article pages, my requests page
> * missing dropdown logo - All community posts in hc/en-us/community/posts
> * missing dropdown logo for Sort by recent activity across user profile pages

These should be fixed now.

> * no indicators of mandatory fields on New Request Form

This is by design (and unrelated to this PR). Optional fields are now marked as `(optional)`

> * settings icon - in community post, comments (articles, posts)

This won't be fixed in this PR. @fredefox is working on that.

> * cross icons on New request page for attachments are not aligned with the file name

I can't recreate this issue.

**Pull request merged**